sdmetrics/reports/single_table/base_report.py

```diff
@@ -0,0 +1,221 @@
+"""Single table base report."""
+import pickle
+import sys
+import warnings
+
+import numpy as np
+import pandas as pd
+import pkg_resources
+import tqdm
+
+from sdmetrics.reports.utils import _validate_categorical_values
```

Comment:
I'm not sure if there is some sort of `pep` or statement in the [google styleguide](https://google.github.io/styleguide/pyguide.html) but if importing a method from other files I tend to make it public (removing the `_` and adding documentation to it.

Reply:
Yes that's a good point. @amontanez24, what would be the best here, making the method public, moving it directly to the `base_report.py` file or keeping it like this?

Reply:
I think moving it to the BaseReport